R/filebacked.R

```diff
@@ -98,3 +98,10 @@ filebacked <- function(scope, path, callback, ...) {
   result
 
 }
+
+renv_filebacked_invalidate <- function(path) {
+  renv_scope_options(warn = -1L)
+  eapply(the$filebacked_cache, function(scope) {
+    rm(list = path, envir = scope)
+  })
```

Comment:
```suggestion
  eapply(the$filebacked_cache, function(envir) {
    rm(list = path, envir = envir)
  })
```

Because it's an environment in an object, not an environment that refers to a frame on the call stack?

Reply:
Good idea. I had reused the name 'scope' here because that's what it's called in `filebacked()`, e.g.

https://github.com/rstudio/renv/blob/35230cd61833a75461e72b06b17f7c01db123be0/R/filebacked.R#L82

but I think that should be called 'context' or 'id' or something separate.